codecov.yml

```diff
@@ -26,16 +33,14 @@ coverage:
     project:
       default:
         informational: true
+
 flag_management:
   # Note: flags should have the same name as the circleci job in which they
   # are uploaded.
   individual_flags:
-    - name: contracts-bedrock-tests
+    - name: contracts-bedrock-coverage
       paths:
         - packages/contracts-bedrock/src
       statuses:
         - type: patch
           target: 100%
-    - name: bedrock-go-tests
```

Comment:
are you sure we should delete `bedrock-go-tests`?

Reply:
That job doesn't appear to do anything other than sit there: https://github.com/ethereum-optimism/optimism/blob/37d7bda6a89aa7d7822debd8c104ae1680c24054/.circleci/config.yml#L1053-L1061

Reply:
Ah we aren't deleting the job, we're just deleting a reference to the job in the Codecov config because that job no longer uploads coverage reports